diff_drive_controller/include/diff_drive_controller/diff_drive_controller.h

```diff
@@ -44,6 +44,9 @@
 #include <pluginlib/class_list_macros.h>
 
 #include <nav_msgs/Odometry.h>
+// @todo add feedback for "cmd_vel_out" (cmd_vel_limited):
```

Comment:
Added. This will allow to check the actual (limited) linear and angular velocities used, which are converted and sent to the wheels.